server-kotlin/src/main/kotlin/main/Application.kt

```diff
@@ -1,86 +1,54 @@
 package main
 
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.joda.JodaModule
-import com.typesafe.config.ConfigFactory
-import digital.cesko.city_request.cityRequestRouter
-import digital.cesko.city_request.google_sheets.GoogleSheets
-import digital.cesko.city_sync.CitySynchronizationService
-import digital.cesko.city_sync.citySynchronizationRouter
-import digital.cesko.city_sync.exception.CitySyncException
+import digital.cesko.city_request.CityRequestController
+import digital.cesko.city_sync.CitySynchronizationController
 import digital.cesko.common.CommonConfig
-import digital.cesko.routers.citySearchRouter
-import io.ktor.application.Application
-import io.ktor.application.call
-import io.ktor.application.install
-import io.ktor.auth.Authentication
-import io.ktor.features.ContentNegotiation
-import io.ktor.features.StatusPages
-import io.ktor.features.XForwardedHeaderSupport
-import io.ktor.http.HttpStatusCode
-import io.ktor.jackson.jackson
-import io.ktor.response.respond
-import io.ktor.routing.routing
-import io.ktor.util.KtorExperimentalAPI
+import digital.cesko.routers.CitySearchController
 import org.jetbrains.exposed.sql.Database
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.event.ApplicationStartedEvent
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.boot.runApplication
+import org.springframework.context.ApplicationListener
+import org.springframework.context.annotation.Bean
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestTemplate
+
+
+@EnableConfigurationProperties(CommonConfig::class)
+// has to enumerate this since we do not follow standard package structure
+@SpringBootApplication(
+    scanBasePackageClasses = [
```

Comment:
Šlo by použít pouze  scanBasePackages = ["digital.cesko"]?

Reply:
Aha, mě nenapadlo, že by někdo udělal balíky jinak než adresáře. Zkusím, co na to Spring.